storage/innobase/os/os0file.cc

```diff
@@ -678,6 +678,13 @@ class AIO {
 		MY_ATTRIBUTE((warn_unused_result));
 #endif /* LINUX_NATIVE_AIO */
 
+	/** Submit buffered AIO requests on the array to the kernel.
+	(low level function).
+	@param[in] acquire_mutex specifies whether to lock array mutex
+	@param[in] array for which to submit IO */
+    static void os_aio_dispatch_read_array_submit_low_for_array(
+		bool acquire_mutex MY_ATTRIBUTE((unused)), AIO* arr);
```

Comment:
Another question: for the reads buffered and not yet submitted by async io, is the IO_FIX of these blocks set to READ?

Reply:
Yes, buf_block_t::page->io_fix = BUF_IO_READ
buf_read_page_low() -> buf_page_init_for_read() -> buf_page_set_io_fix()

Reply:
Thanks for checking